### DISCUSSION TITLE
myst parser for sphinx documentation

### DISCUSSION THREAD

**Discussion by nathanrpage97** [Textualize/rich#1280] [Ideas]
What are your thoughts on including the [myst parser](https://myst-parser.readthedocs.io/en/latest/) to enable docs to be written in markdown? I believe it makes documentation writing more accessible to other users. I have used it in my own work and found it much easier.

[There is a great pycon 2021 talk going over it](https://www.youtube.com/watch?v=YclYtM56qjo)

**Comment by willmcgugan** [comment]
I've never been a big fan of the RST syntax, so in principle I'm for it. TBH though it looks like once you've introduced all those directives then the myst syntax is not all that different.

**Reply by nathanrpage97** [reply]
The other option is using MkDocs which has some other nice features, but that would require a much larger change to  the docs all at once. I thought it would be good to use the myst parser and slowly convert everything to markdown. Then it would make a possible conversion to MkDocs much easier if desired.
